install is broken for 'cargo install sccache --no-default-features --features=redis'

When trying to build sccache with redis support and without default features, I get the following error message.

```
   Compiling sccache v0.3.0 (registry `...`)
error[E0432]: unresolved import `url`
  --> C:\cargo\registry\src\artifactory.paesslergmbh.de-f98ca9d7d26c5c3b\sccache-0.3.0\src\cache\redis.rs:23:5
   |
23 | use url::Url;
   |     ^^^ use of undeclared crate or module `url`
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -56,11 +56,11 @@ number_prefix = "0.4"
 openssl = { version = "0.10", optional = true }
 percent-encoding = { version = "2", optional = true }
 rand = "0.8.4"
-redis = { version = "0.21", optional = true, default-features = false, features = ["aio", "tokio-comp", "tokio-native-tls-comp"] }
 regex = "1"
 reqwest = { version = "0.11", features = ["json", "blocking", "stream"], optional = true }
 retry = "1"
 ring = { version = "0.16", optional = true, features = ["std"] }
+rredis = { package = "redis", version = "0.21", optional = true, default-features = false, features = ["aio", "tokio-comp", "tokio-native-tls-comp"] }
 semver = "0.9"
 sha-1 = { version = "0.10.0", optional = true }
 sha2 = { version = "0.10.1", optional = true }
@@ -126,6 +126,7 @@ simple-s3 = []
 gcs = ["chrono", "hyper", "hyperx", "percent-encoding", "reqwest", "ring", "url"]
 memcached = ["memcached-rs"]
 native-zlib = []
+redis = ["url", "rredis"]
 # Enable features that require unstable features of Nightly Rust.
 unstable = []
 # Enables distributed support in the sccache client

--- a/src/cache/redis.rs
+++ b/src/cache/redis.rs
@@ -17,6 +17,7 @@ use crate::cache::{Cache, CacheRead, CacheWrite, Storage};
 use crate::errors::*;
 use redis::aio::Connection;
 use redis::{cmd, Client, InfoDict};
+use rredis as redis;
 use std::collections::HashMap;
 use std::io::Cursor;
 use std::time::{Duration, Instant};

